From c8d13f31e0956842c36af781ceb3b7fd6daa6f73 Mon Sep 17 00:00:00 2001
From: Philip Balister <philip@balister.org>
Date: Fri, 5 Aug 2016 13:39:15 -0400
Subject: [PATCH 2/2] Fix OpenEmbedded builds for targets that use lib64.

Use CMAKE_INSTALL_LIBDIR to determine if the target uses lib64.

Signed-off-by: Philip Balister <philip@balister.org>
---
 CMakeLists.txt                 | 2 +-
 cmake/Modules/GrPlatform.cmake | 7 +++++++
 2 files changed, 8 insertions(+), 1 deletion(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 1f7179c..764ab79 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -155,5 +155,5 @@ add_subdirectory(docs)
 # Install cmake search helper for this library
 ########################################################################
 install(FILES cmake/Modules/framersConfig.cmake        
-    DESTINATION lib/cmake/framers
+    DESTINATION lib${LIB_SUFFIX}/cmake/framers
 )
diff --git a/cmake/Modules/GrPlatform.cmake b/cmake/Modules/GrPlatform.cmake
index 0e9891a..96c2bbf 100644
--- a/cmake/Modules/GrPlatform.cmake
+++ b/cmake/Modules/GrPlatform.cmake
@@ -38,6 +38,13 @@ if(NOT CMAKE_CROSSCOMPILING AND LINUX AND EXISTS "/etc/redhat-release")
 endif()
 
 ########################################################################
+# Detect /lib versus /lib64
+########################################################################
+if (CMAKE_INSTALL_LIBDIR MATCHES lib64)
+    set(LIB_SUFFIX 64)
+endif()
+
+########################################################################
 # when the library suffix should be 64 (applies to redhat linux family)
 ########################################################################
 if(NOT DEFINED LIB_SUFFIX AND REDHAT AND CMAKE_SYSTEM_PROCESSOR MATCHES "64$")
-- 
2.5.5

